RELEASE_NOTES.md

```diff
@@ -13,6 +13,10 @@ v1.0.0-alpha.x
 - Removed `toml++` as vendored library. Is now an external dependency
   similar to other external dependencies.
 
+- Moved `setup.py` and `pyproject.toml` under the Python component's
+  directory, i.e. `src/openassetio-python`.
```

Comment:
Could/should it go [here](https://github.com/OpenAssetIO/OpenAssetIO/blob/22b434d937a1d7f66b7d9bfbe886599a7bcce196/src/openassetio-python/pyproject.toml#L6) too?

Reply:
> Could/should it go [here](https://github.com/OpenAssetIO/OpenAssetIO/blob/22b434d937a1d7f66b7d9bfbe886599a7bcce196/src/openassetio-python/pyproject.toml#L6) too?

As mentioned in https://github.com/OpenAssetIO/OpenAssetIO/pull/754#discussion_r1028282735 - if you put `pip` in there then that'll be installed as a package, but won't be the pip that is actually used (if you think about it, it's a bit chicken-and-egg)

Reply:
Maybe worth a note in the release notes, and perhaps a slight rewording of the text - as I got the exception that it couldn't find CMAKE lists that only talks about wheels when I tried with an old version?

Reply:
Ah OK, yeah I see that could be confusing.  I'll just remove the "working directory" bit from the error message, since it's really advanced/unsupported usage to install using anything other than pip.

Reply:
Ahh does the "invocation magic" cd you in there with you use `python -m pip`? If that is the case then the `__file__` thing I was waffling about below seems a little redundant.

Reply:
Updated release notes and exception message in 99cce633